src/test/mon/test-mon-msg.cc

```diff
@@ -224,6 +224,7 @@ class MonMsgTest : public MonClientHelper,
   Cond cond;
 
   MonMsgTest() :
+    reply_msg(NULL),
```

Comment:
could also initialize it using in-class initialization, see http://en.cppreference.com/w/cpp/language/data_members